### DISCUSSION TITLE
[QUESTION] When will arclight 1.21 release

### DISCUSSION THREAD

**Discussion by OdymarGamer** [IzzelAliz/Arclight#1503] [Q&A]
Hi i was wondering when a stable version of 1.21 will release i dont mean to be the guy that spams for eta 
but i was wondering if maybe you had a release date

**Comment by qyl27** [comment]
Sending questions in the discussion would be better, issues were used for tracking the bugs or crashes, that I will convert this into a discussion.  
We couldn't provide an exact stable release date because we are not full-time in maintenance.  
Builds in actions page are available, if you have any issues, feel free to submit an issue.

**Comment by EOSRX24** [comment]
I would like to know too.
